src/public/infernode/components/layout/NavBar/NavBar.tsx

```diff
@@ -45,7 +45,7 @@ export default function NavBar(): JSX.Element {
               <Nav.Link>Capture</Nav.Link>
             </LinkContainer>
             <LinkContainer to="/manage">
-              <Nav.Link>Manage</Nav.Link>
+              <Nav.Link>Upload</Nav.Link>
```

Comment:
It looks like `src/__tests__/navbarUnit.test.tsx` needs to be updated with the new navBar page name as well.

Reply:
Gotcha.